src/server/routes/api/index.js

```diff
@@ -119,27 +119,27 @@ function constructDateQuery(dateInput, isEndDate = false) {
   const lastSecond = (isEndDate) ? 'T23:59:59' : '';
 
   if (dateInput && typeof dateInput === 'string') {
-    const dateArray = dateInput.split('/');
+    const dateArray = dateInput.split('-');
 
     // Checks if it has a valid date format. The Regex check if the inputs are digits
     // and if they have right number of digits
-    const date_matches = dateInput.match(/^(\d{2})\/(\d{2})\/(?:\d{4})$/);
+    const date_matches = dateInput.match(/^(\d{4})\-(\d{2})\-(?:\d{2})$/);
 
     if (!date_matches) {
       return undefined;
     }
 
     // Checks if the month is valid
-    if (parseInt(dateArray[0], 10) < 1 || parseInt(dateArray[0], 10) > 12) {
+    if (parseInt(dateArray[1], 10) < 1 || parseInt(dateArray[1], 10) > 12) {
```

Comment:
Ha ha! See, if you'd had `month` and `year` as variables you could have saved 2 or 3 keystrokes ;-)

Reply:
[Callback to previous PR in case anyone else is confused]

Reply:
Updated it in the previous PR haha